easybuild/easyblocks/g/gromacs.py

```diff
@@ -70,6 +70,26 @@ def __init__(self, *args, **kwargs):
         self.lib_subdir = ''
         self.pre_env = ''
 
+        # check whether PLUMED is listed as a dependency
+        self.with_plumed = 'PLUMED' in [dep['name'] for dep in self.cfg['dependencies']]
```

Comment:
Well, we can certainly do better than `get_software_root`, yes, but it's not implemented in framework yet...